packages/dd-trace/src/opentracing/tracer.js

```diff
@@ -54,7 +56,21 @@ class DatadogTracer {
 
     // as per spec, allow the setting of service name through options
     const tags = {
-      'service.name': options?.tags?.service ? String(options.tags.service) : this._service
+      'service.name': options?.tags?.service ? String(options.tags.service) : this._service,
+      ...options?.tags
+    }
+
+    if (options?.kind) {
+      tags['span.kind'] = options.kind
+    }
+
+    options.tags = tags
```

Comment:
Why all these changes to tags, on lines 59-67?

Reply:
Leftover from an initial implementation I did, they are not necessary so I have removed these changes.